### PR TITLE
gwyddion: update to 2.62

### PR DESCRIPTION
**Pull request by rowue**
#### Description

Update of gwyddion to 2.62

###### Type(s)

- [ ] bugfix
- [X] enhancement
- [ ] security fix

###### Tested on

macOS 10.13.6 17G14042 x86_64
Xcode 10.1 10B61 / Command Line Tools x.y.z

###### Verification 
Have you

- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [X] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`? 
  - there aren't any tests
- [ ] tried a full install with `sudo port -vst install`?
  - took more than 24hrs - canceld therefore
  - 'sudo port -v install gwyddion' worked without a charm on a clean installation
- [X] tested basic functionality of all binary files?
- [X] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: science/gwyddion/Portfile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 PortSystem          1.0
 
 name                gwyddion
-version             2.59
+version             2.62
 revision            1
```
Comment (reneeotten) [inline science/gwyddion/Portfile:7]: revision should be reset to zero when updating the version

**Review by reneeotten — CHANGES_REQUESTED** [review]
also please make sure to rebase with master and squash your commits.

File: science/gwyddion/Portfile (hunk 1 above)
Comment (reneeotten) [reply science/gwyddion/Portfile]: please don't comment out the line, just set it to zero as I said before

Comment (rowue) [reply science/gwyddion/Portfile]: Now it's zero

**Review by reneeotten — CHANGES_REQUESTED** [review]
CI still fails, but that's appears to be an issue that cannot be easily solved on current macOS version due to the build failure of `pangox-compat`

**Review by reneeotten — APPROVED** [review]

**Pull request merged**